content/post/know-your-aws-community-day-india-2020-speaker-swapnil-jakhetia.md

```diff
@@ -0,0 +1,20 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T14:12:08Z
+description = "Swapnil Jakhetia will be speaking on \"Infrastructure is Code with AWS CDK\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/swapnil.PNG"
+tags = ["AWS", "AWSCommunityDayIndia", "AWS CDK", "Speaker"]
+title = "Know Your AWS Community Day India 2020 Speaker - Swapnil Jakhetia"
+
++++
+Swapnil Jakhetia is working as a Solution Architect at Blazeclan, India will be speaking on "**Infrastructure is Code with AWS CDK**" at <time> on <date> at AWS Community Day India 2020.
```

Comment:
date/time placeholders are missing data. You can find the timing details from: 

https://communityday.awsug.in/

Reply:
Corrected!